cppsrc/U8x8lib.cpp

```diff
@@ -45,6 +45,15 @@
 #include <Wire.h>
 #endif
 
+/* ISSUE #991 */
+#if defined(STM32F1) || defined(STM32F1xx) || defined(STM32F4) || defined(STM32F4xx)
+SPIClass U8G2_SPI2(2);
+bool First_Time = true;
+#else
+SPIClass U8G2_SPI2 = SPI2;
```

Comment:
Not so nice, this will make a copy of the complete structure, wasting that much memory in RAM.

Reply:
One more point: at this time, we do not know whether SPI2 exists.

Reply:
In fact the second SPI is called SPI1. This will break on almost all none STM32 plattforms. I can not merge this,

Reply:
Oooops i've made a mistake obviusly i've to write SPI1 and not SPI2. 
I know that this can take more Memory but in my SPI lib in which i'll open an issue too, SPI1 is not from SPIClass.